lib/NativeCall.pm

```diff
@@ -22,11 +22,11 @@ sub _attr_parse {
     (\w+)
     (?:
       \(
-      (.*?)
+      (.*)
```

Comment:
Right.  It needs to be greedy in order to capture the closure specification (int)->int.  Without the greedy you get these failures with the test that I constructed:

```
crux% prove -lv t/closure.t 
t/closure.t .. 
not ok 1

#   Failed test at t/closure.t line 18.
#     Structures begin differing at:
#          $got->[0] = '(int'
#     $expected->[0] = '(int)->int'
not ok 2

#   Failed test at t/closure.t line 21.
#     Structures begin differing at:
#          $got->[0] = '(int'
#     $expected->[0] = '(int)->int'
not ok 3

#   Failed test at t/closure.t line 24.
#     Structures begin differing at:
#          $got->[0] = '(int'
#     $expected->[0] = '(int,int)->int'
not ok 4

#   Failed test at t/closure.t line 27.
#     Structures begin differing at:
#          $got->[0] = '(int'
#     $expected->[0] = '(int,int)->int'
1..4
# Looks like you failed 4 tests of 4.
Dubious, test returned 4 (wstat 1024, 0x400)
Failed 4/4 subtests 

Test Summary Report
-------------------
t/closure.t (Wstat: 1024 Tests: 4 Failed: 4)
  Failed tests:  1-4
  Non-zero exit status: 4
Files=1, Tests=4,  0 wallclock secs ( 0.03 usr  0.01 sys +  0.04 cusr  0.01 csys =  0.09 CPU)
Result: FAIL
```